From 7520d243ed4db53a8f29ce92b42907e0c8033f88 Mon Sep 17 00:00:00 2001
From: Alberto Bursi <bobafetthotmail@gmail.com>
Date: Mon, 8 Jun 2020 19:59:35 +0200
Subject: [PATCH 2/3] hwinfo: disable compile with uclibc

this package fails to compile with uclibc,
disable it if the uclibc toolchain is selected

Signed-off-by: Alberto Bursi <bobafetthotmail@gmail.com>
(cherry picked from commit 669dddef24f2c52ed3f4ace339be34da9c9fe18f)
---
 utils/hwinfo/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/hwinfo/Makefile b/utils/hwinfo/Makefile
index d99a05368..347d3488d 100644
--- a/utils/hwinfo/Makefile
+++ b/utils/hwinfo/Makefile
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hwinfo
 PKG_VERSION:=21.70
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/openSUSE/hwinfo/tar.gz/$(PKG_VERSION)?
@@ -65,7 +65,7 @@ define Package/hwinfo
   CATEGORY:=Utilities
   TITLE:=probe the hardware present in the system
   URL:=https://github.com/openSUSE/hwinfo
-  DEPENDS:= +libuuid
+  DEPENDS:= +libuuid @!USE_UCLIBC
 endef
 
 define Package/hwinfo/description
-- 
2.25.1

